addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection_tests.js

```diff
@@ -91,6 +91,56 @@ QUnit.test('select another mailbox', async function (assert) {
     );
 });
 
+QUnit.test('navigate to inbox tab when discuss is loaded from another tab than mailbox (like when user come from a notification)', async function (assert) {
+    assert.expect(5);
+
+    this.data['mail.channel'].records.push({ id: 20 });
+    this.data['mail.message'].records.push({
+        channel_ids: [20],
+        date: "2019-04-20 10:00:00",
+        id: 100,
+        model: 'mail.channel',
+        res_id: 20,
+    });
+
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 20,
+            },
+        }
+    });
+
+    assert.hasClass(
+        document.querySelector('.o_MobileMessagingNavbar_tab[data-tab-id="channel"]'),
+        'o-active',
+        "the 'channel' tab should be the active tab"
```

Comment:
_"'channel' tab should be active initially when loading discuss with channel id as active_id"_